### PR TITLE
Fix categories page

### PR DESCRIPTION
**Pull request by Aliiiu**
# Fix categories page inconsistency
This PR addresses the inconsistency in the categories index route and sub-route. 

image

image

closes https://github.com/EveripediaNetwork/issues/issues/1681

### CODE REVIEW

**Comment by Aliiiu** [comment]
> #F9FAFB

@Simplypheyie96 

image

image

File: src/components/Landing/AboutIqgpt.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  Image,
+  SimpleGrid,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import React from 'react'
+import { LinkButton, Logo } from '../Elements'
+import { useTranslation } from 'react-i18next'
+
+const CustomListItem = ({ text }: { text: string }) => {
```
Comment (Adebesin-Cell) [inline src/components/Landing/AboutIqgpt.tsx:16]: Give a more descriptive name.

**Review by Adebesin-Cell — COMMENTED** [review]
Weird tho, why's ``Landing`` hero section PR in this PR, makes the review confusing.

File: src/components/Landing/AboutIqgpt.tsx (hunk 1 above)
Comment (Aliiiu) [reply src/components/Landing/AboutIqgpt.tsx:16]: will resolve that

Comment (Aliiiu) [reply src/components/Landing/AboutIqgpt.tsx]: you can check now

**Comment by Adebesin-Cell** [comment]
One thing, if you made changes requested, you can ask for a re-review of the PRs.

File: src/components/Landing/TrendingCard.tsx [hunk 2]
```diff
@@ -87,9 +92,23 @@ const TrendingCard = ({
...
+                  borderRadius: 'md',
+                  bgColor: 'gray.100',
+                  _dark: {
+                    bgColor: 'whiteAlpha.100',
+                  },
+                }}
+                cursor="pointer"
+                mt={0}
+                key={i}
+                py={2}
```
Comment (Adebesin-Cell) [inline src/components/Landing/TrendingCard.tsx:109]: Can be 
```suggestion
                p={2}
```

**Comment by Aliiiu** [comment]
> One thing, if you made changes requested, you can ask for a re-review of the PRs. Also, instead of having so many PRs opened, ensure one issue is completed before moving to another. If there's a blocker you can reach out to @Softdev1 or @Royal-lobster to get unblocked.

Completed the task but the design still needed adjustment. I had to wait for her to complete the design. I was open to more issues so I requested more.

**Comment by Adebesin-Cell** [comment]
> > One thing, if you made changes requested, you can ask for a re-review of the PRs. Also, instead of having so many PRs opened, ensure one issue is completed before moving to another. If there's a blocker you can reach out to @Softdev1 or @Royal-lobster to get unblocked.
> 
> Completed the task but the design still needed adjustment. I had to wait for her to complete the design. I was open to more issues so I requested more.

I see. Alright

**Review by Simplypheyie96 — APPROVED** [review]

**Pull request merged**